vms/about/templates/about/about.html

```diff
@@ -0,0 +1,32 @@
+{% extends "vms/base.html" %}
+
+{% load i18n %}
+
+{% block content %}
+<div  class="medium">
+
+<h1>History</h1>
+<hr>
+<p>Systers has many volunteers and potential volunteers that offer their skills, time and commitment to our vision. This Volunteer Management System will be particularly useful for volunteer coordinators, volunteer sign-up, tracking hours, and reporting.</p>
```

Comment:
Bump. (´･_･`)

All this won't be going in the about page. If I am not misremembering we decided to not have a about page, also updated the issue. Can you please link the issue, here?

Reply:
Hey ! So , what is to be included in the about page? Can you suggest? Are we not supposed to implement the about page as in #403 ?